tests/hardhat/Fork/constants.ts

```diff
@@ -0,0 +1,92 @@
+export const archiveNodes = {
```

Comment:
I would unify these env variables, as we are doing for example [here](https://github.com/VenusProtocol/vips/blob/develop/.env.example#L8)

Reply:
Done https://github.com/VenusProtocol/isolated-pools/commit/167b53cc4b84708de40e36a43a0a9094ca706b0d